modin/pandas/series.py

```diff
@@ -683,6 +683,31 @@ def factorize(self, sort=False, na_sentinel=-1):
             pandas.Series.factorize, sort=sort, na_sentinel=na_sentinel
         )
 
+    def fillna(
+        self,
+        value=None,
+        method=None,
+        axis=None,
+        inplace=False,
+        limit=None,
+        downcast=None,
+    ):
+        if isinstance(value, BasePandasDataset) and not isinstance(value, Series):
+            raise TypeError(
+                '"value" parameter must be a scalar or dict, but '
```

Comment:
```suggestion
                '"value" parameter must be a scalar, dict or Series, but '
```

Reply:
Ok

Reply:
You didn't fix this.

Reply:
Yes, apparently I forgot to add series.py file to index when I committed.